modules/vehicles/api/vehicles-routes.ts

```diff
@@ -0,0 +1,13 @@
+import { Request, Response } from 'express';
+import * as core from 'express-serve-static-core';
+import { getQtyFromRequest } from '../../../utils/route-utils';
+import getRandomVehicles from '../utils/getRandomVehicles';
+
+module.exports = function (app: core.Express) {
+  //Get a randomly generated user
+  app.get('/vehicles/:qty?', (req: Request, res: Response) => {
```

Comment:
We no longer are manually updating swagger.json as such please can you add OpenAPI comments for the api endpoint here and also in your vehicle.ts file.

Reply:
Secondly can you add tests for the end point please, examples of which exist in other modules. 
I have updated the target branch of this merge to v0,13.0 which is where the latest code lives. 👍 

Thanks for the hard work

Reply:
Thanks, this PR is now updated @ageddesi